src/analyses/libraryFunctions.ml

```diff
@@ -88,6 +96,43 @@ let posix_descs_list: (string * LibraryDesc.t) list = LibraryDsl.[
     ("getsockopt", unknown [drop "sockfd" []; drop "level" []; drop "optname" []; drop "optval" [w]; drop "optlen" [w]]);
     ("gethostbyaddr", unknown [drop "addr" [r_deep]; drop "len" []; drop "type" []]);
     ("gethostbyaddr_r", unknown [drop "addr" [r_deep]; drop "len" []; drop "type" []; drop "ret" [w_deep]; drop "buf" [w]; drop "buflen" []; drop "result" [w]; drop "h_errnop" [w]]);
+    ("sigaction", unknown [drop "signum" []; drop "act" [r_deep; s_deep]; drop "oldact" [w_deep]]);
+    ("tcgetattr", unknown [drop "fd" []; drop "termios_p" [r_deep]]);
```

Comment:
Is `r_deep` right here for `termios_p`? 

> The tcgetattr() function shall get the parameters associated with
       the terminal referred to by fildes and store them in the termios
       structure referenced by termios_p.

Reply:
Indeed, I must've mixed up `tcgetattr` and `tcsetattr`.